blocks/core/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.xml

```diff
@@ -1 +1,7 @@
-<item title="{{title | default('Container Title')}}" more-page-text="{{pagetext}}" more-page-link="{{ pagelink | default('#') }}" show-footer="{{ showfooter | default('yes') }}" modifiers="{{ modifiers }}"/>
+<item title="{{title | default('Container Title')}}" more-page-text="{{pagetext}}" more-page-link="{{ pagelink | default('#') }}" show-footer="{{ showfooter | default('yes') }}" modifiers="{{ modifiers }}">
+	{% if modules %}
```

Comment:
I think it makes more sense for this to be a single node named something like `content` rather than a list of `module`s as it'll always be a single node being passed in to the container.